### PR TITLE
New post: A Gentle Introduction to Transformer Architecture and Relevance to Generative AI

### PR DESCRIPTION
**Pull request by Xiaoyuao988**
# Content Pull Request Checklist

Please check to make sure your content submission meets the following requirements by adding an `x` between the brackets:

- [X ] The title of this pull request reflects the title of my content.
- [X ] I have reviewed the [Content Review Checklist](https://github.com/build-on-aws/content/blob/main/CONTENT_REVIEW_CHECKLIST.md).
- [X ] This pull request contains one piece of content (post or page).

Is this pull request linked to a Content Proposal or Idea Suggestion Issue? If so, please link to it below using the number or full link.

```
This post is Topic #34 on the Content Plan here:
https://quip-amazon.com/pKNoAdz3MEIy/Content-Plan-for-2023-Q2-Data-Eng-DataOps-ML

```

**By submitting this pull request, I confirm I own this content, have permission to use each image included in this content, and that it will be released under the [CC BY-SA 4.0 SA License](/LICENSE).**

### CODE REVIEW

File: posts/a-gentle-intro-to-transformer-and-gen-ai/index.md [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+   1. Supervised fine tuning (SFT).
+   2. Reward Model training, based on human feedback. Basically define what "good" looks like.
+   3. Reinforcement Learning on this Reward Model. This part is to further fine-tuning the model.
+
+Human feedback would be subjective, but what RLHF doing well is to aggregate human preferences at scale.
+
+One example use case of RLHF with LLM is de-toxicity of outputs. 
+
+### Summary - it is all about Customization and Optimization
+
```
Comment (LindaHaviv) [inline posts/a-gentle-intro-to-transformer-and-gen-ai/index.md:272]:
```suggestion
n conclusion, customization of pre-trained LLM models to fit your specific needs is an evolving field with numerous ongoing research efforts; hopefully, the popular techniques shared here offer valuable guidance as you embark on this exciting journey.
```

**Review by LindaHaviv — CHANGES_REQUESTED** [review]
Great article! Made some minor suggested edits. Awesome work.

File: posts/a-gentle-intro-to-transformer-and-gen-ai/index.md (hunk 1 above)
Comment (LindaHaviv) [reply posts/a-gentle-intro-to-transformer-and-gen-ai/index.md:272]: Also this is just a suggested sentence. Anything that just ties in the summary would help flow.

Comment (LindaHaviv) [reply posts/a-gentle-intro-to-transformer-and-gen-ai/index.md:272]: Another suggestion is to add a conclusion sentence at the end of the entire article

Comment (Xiaoyuao988) [reply posts/a-gentle-intro-to-transformer-and-gen-ai/index.md:272]: I have updated this part.

**Review by LindaHaviv — APPROVED** [review]

**Pull request merged**